tests/python/frontend/onnx/test_forward.py

```diff
@@ -3628,7 +3628,6 @@ def verify_roi_align(
     test_clip_min_max_as_inputs()
     test_onehot()
     test_matmul()
-    test_batch_matmul()
```

Comment:
Just wanted to note that I removed this since `test_batch_matmul` is now run with `tvm.testing.parametrize`, which means it will cause an error when run using python instead of pytest.